modules/reference/hardware/lenovo-x1/definitions/x1-gen11.nix

```diff
@@ -99,6 +99,15 @@
   #   00:1f.x in the example from Lenovo X1 Carbon
   #   must be defined for passthrough to AudioVM
   audio = {
+    # Force a PCI device reset to the audio device
+    # This is to get the pci hardware device to the default state at shutdown
+    removePciDevice = "0000:00:1f.3";
```

Comment:
I think the audio device PCI id could be anything but on our current target Lenovo and Dell devices it just happens to be the same.
I'm not sure if other devices besides Lenovo X1 require resetting the audio PCI device to get it back to working order on host (or audiovm in case of restart). If the restart of the pci device is not required on some platforms the value can be left unset so the reset is skipped.

I guess the point is that this should be tested with Dell and set the Dell configuration to match if acpi and reset is required (or not).